package.json

```diff
@@ -32,6 +32,14 @@
     "test-multibrowser": "grunt test:multibrowser",
     "test-snyk": "snyk test"
   },
+  "files": [
+    "bin",
+    "build",
+    "lib",
+    "index.js",
```

Comment:
However it includes a lot of not used files and would be only important if we would enable source maps